starttls_policy/tests/configure_test.py

```diff
@@ -86,5 +122,24 @@ def test_instruct_string(self):
         self.assertTrue("postfix reload" in instructions)
         self.assertTrue(generator.default_filename in instructions)
 
+def enclose_config_case(conf, expected):
+    """Wrap testcase against config into closure in order to make it attachable
+    to main testcase class"""
+    def test(self):
+        with TempPolicyDir(conf) as testdir:
+            generator = configure.PostfixGenerator(testdir)
+            generator.generate()
+            pol_filename = os.path.join(testdir, generator.default_filename)
+            with open(pol_filename) as pol_file:
+                result = pol_file.read()
+            os.remove(pol_filename)
+        self.assertEqual(result, expected)
+    return test
+
+for tname, a, b in testgen_data:
+    test_name = "test_%s" % tname
+    wrapped_test = enclose_config_case(a, b)
```

Comment:
naming nit: prefer longer, detailed names: `config_name, config_content, expected_output in testgen_data`